packages/amazon-sumerian-hosts-babylon/src/Babylon.js/PointOfInterestFeature.js

```diff
@@ -23,6 +23,10 @@ class PointOfInterestFeature extends CorePointOfInterestFeature {
     return obj instanceof TransformNode;
   }
 
+  /**
+   *
```

Comment:
nit: Do we need a description for the function here? If not, I think we can remove line 27.